cpp/src/replace/replace.cu

```diff
@@ -53,16 +53,27 @@ namespace{ //anonymous
                       gdf_size_type                      nrows,
                       thrust::device_ptr<const T> old_values_begin,
                       thrust::device_ptr<const T> old_values_end,
-                      const T*                    d_new_values)
+                      const T*                    d_new_values,
+                      gdf_valid_type*             col_valid,
```

Comment:
For performance use `bit_mask_t` and cast the gdf_valid_type pointer to bit_mask_t in the calling function.

Reply:
changed to `bit_mask_t`, as suggested.